openwisp_controller/config/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from django_netjsonconfig.utils import get_api_urls
+from openwisp_controller.config.api import views
+
+app_name = 'openwisp_controller'
+
+urlpatterns = [
+    url(r'^api/v1/templates/create/$',
```

Comment:
the url path should be `r'^api/v1/templates/$'`, this URL must be able to do both listing for `GET` requests and create new templates for `POST` requests